decidim/decidim
# Add moderations to Open Data files

#### :tophat: What? Why?
This PR adds moderation data available to the OpenData files. 

#### :pushpin: Related Issues
*Link your PR to an issue*
- Fixes #13529

#### Testing
1. Switch to the branch
2. Start app, and export the Open data
3. See the archive contains moderations.csv & moderated_users.csv
4. See there is a proper Readme in place

### :camera: Screenshots
*Please add screenshots of the changes you are proposing*
![Description](URL)

:hearts: Thank you!
